### DISCUSSION TITLE
idea for solution without Wifi? connect sensors directly to raspberry pi?

### DISCUSSION THREAD

**Discussion by sir106** [klaasnicolaas/home-assistant-glow#181] [Q&A] [answered]
I really like this project - but where my power meter is installed, there is absolutely NO wifi signal available. I have only ethernet available downstairs.

already installed my home assistant instance on an raspberry PI and connected via ethernet cable - is there a way to connect the sensors directly to the raspberry?

**Comment by klaasnicolaas** [comment, marked as answer]
What if you expand your WiFi network via a powerline with WiFi repeater?

**Reply by sir106** [reply]
thanks! that was also my first idea - but I now solved it wiring my meter directly to the raspberry and using the edl21 integration (https://www.home-assistant.io/integrations/edl21/) which is very lightweight approach... 

no additional devices needed ;)
